src/MovieInfo/MovieInfo.css

```diff
@@ -10,19 +19,25 @@
 }
 /* ~~~ LEFT CONTAINER ~~~ */
 
-.movie-info {
-  display: flex;
+.movie-info-left {
+  /* display: flex;
   flex-direction: column;
-  text-align: center;
+  justify-content: space-between; */
+  /* text-align: center; */
   width: 30%;
-  height: fit-content;
+}
```

Comment:
Whats the difference between movie-info-left and movie-info? May want to think of a better naming convention here?